samples/wifi/scan/Kconfig

```diff
@@ -33,4 +33,38 @@ config WIFI_SCAN_TYPE_PASSIVE
 	help
 	  Enable passive scan
```

Comment:
Instead of "Enable passive scan" say "Force passive scan on all channel"

Reply:
Similarly for "Active scan" instead of "Enable active scan" say "Force use of active scan for all channels where active scan is permitted by regulatory domain restrictions"